index.js

```diff
@@ -226,6 +228,15 @@ const main = async () => {
   core.setOutput('undefined', indeterminate.toString())
   core.setOutput('ignored', ignored.toString())
   core.setOutput('total', total.toString())
+  core.startGroup('Findings')
```

Comment:
What are your thoughts on either hiding the findings group if `findingsDetails.length === 0` or change the log output inside the group to `No findings`.

Reply:
I feel that if there are 0 findings, we should not show a group output at all.  Albeit sounds odd, but someone may want to evaluate the log output and use each line in the "Findings" group to do something.. 

Perhaps this request can be forked as an issue / separate enhancement and more discussion there?  I feel we have a minimum viable product here, which will provide additional value, and is ready to ship.